src/Makefile.am

```diff
@@ -19,4 +19,4 @@ endif
 
 bin_PROGRAMS = patchelf
 
-patchelf_SOURCES = patchelf.cc elf.h
+patchelf_SOURCES = patchelf.cc elf.h patchelf.h
```

Comment:
@Mic92 found the reason!
I am not very familiar with autotools, so this is all new to me :)